Runtime/Model/BacktraceHttpClient.cs

```diff
@@ -47,7 +47,11 @@ public void Post(string submissionUrl, BacktraceJObject jObject, Action<long, bo
             asyncOperation.completed += (AsyncOperation operation) =>
             {
                 var statusCode = request.responseCode;
-                var isNetworkError = request.isNetworkError || request.isHttpError;
+#if UNITY_2020_2_OR_NEWER
+                var isNetworkError = request.result == UnityWebRequest.Result.ConnectionError || request.result == UnityWebRequest.Result.ProtocolError;

```

Comment:
We might need to move `var isNetworkError` outside of this #ifdef, otherwise the #else condition doesn't assign anything